### DISCUSSION TITLE
Support for custom 3D meshes? Is this even possible to make based on this library?

### DISCUSSION THREAD

**Discussion by tkgalk** [StarArawn/bevy_ecs_tilemap#302] [Q&A]
https://catlikecoding.com/unity/tutorials/hex-map/part-1/ <- essentially this. I'm wondering if instead of working my way through building the grid and the mesh myself is there a possibility to utilise this library for the grid part while constructing a custom mesh.

Any ideas?

**Comment by bzm3r** [comment]
Yes, I think this should be possible. The code for handling the grids lies in [`helpers`](https://github.com/StarArawn/bevy_ecs_tilemap/tree/main/src/helpers), and it is fairly independent of the rendering stuff. If you don't want to render things, you just [don't need to insert tile bundles for each tile](https://github.com/StarArawn/bevy_ecs_tilemap/blob/eabd66eac4c5204ec54646141ffb26935cace88d/examples/basic.rs#L37). 

I would suggest you browse the code for hex grids in `helpers`, and maybe use the pieces that you need.
